index.js

```diff
@@ -478,23 +483,46 @@ function modifyFirstrun() {
     });
 }
 
+/**
+ * Overrides time interval defauls from config file.
+ */
+function overrideDefaults() {
+    try {
+        // load the config file
+        let config = self.data.load('./config.json');
```

Comment:
Oh! I know what happened. When adding the file to `.jpmignore`, it is doing exactly that when building the XPI :) Ha! I will update the default location and documentation to reflect this change.